integrated_channels/utils.py

```diff
@@ -235,6 +235,22 @@ def get_duration_from_estimated_hours(estimated_hours):
     return None
 
 
+def get_courserun_duration_in_days(course_run):
+    """
+    Return the duration in number of days corresponding to a course run.
+    """
+    min_effort = course_run.get('min_effort')
+    max_effort = course_run.get('max_effort')
+    weeks_to_complete = course_run.get('weeks_to_complete')
+    if min_effort and max_effort and weeks_to_complete:
+        hours_per_day = 8.0
```

Comment:
Nit: could be a constant (otherwise I like this function, it's very clear and concise).